### PR TITLE
Add videojs-hotkeys@0.2.18 w/ git auto-update

### PR DESCRIPTION
**Pull request by sophie-li**
Pull request for issue: #9818 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ctd1500/videojs-hotkeys
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/videojs-hotkeys
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 6
   - Count of stars: 65
   - Count of forks: 48
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/videojs-hotkeys/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ctd1500/videojs-hotkeys.git"
+  },
+  "keywords": [
+    "videojs",
+    "videojs-plugin",
+    "hotkeys"
+  ],
```
Comment (sufuf3) [inline ajax/libs/videojs-hotkeys/package.json]: Would you add `"videojs"` in keywords?
I found it show here https://github.com/ctd1500/videojs-hotkeys/blob/master/bower.json#L15

Add more keyword can let people search the library more easier.

**Comment by sufuf3** [comment]
Hi @sophie-li Do you want to continue to work on this?
Thank you.

**Comment by sophie-li** [comment]
@sufuf3 hi, sorry for the late reply! please take over this task. thanks

**Comment by sufuf3** [comment]
@PeterDaveHello should we help @sophie-li to fix this PR and let this PR better to merge.
Or close this PR let the beginner try this related issue?
Thank you.

**Comment by PeterDaveHello** [comment]
Let's close it, we can open it again when @sophie-li has time to finish it.

**Pull request closed without merging**